syntax = "proto3";

package local;

import "redis.proto";

message Online {
    repeated redis.Stats stats = 1;
}

message History {
    repeated redis.Stats stats = 1;
}

message LocalStats {
    Online online = 1;
    History history = 2;
    int64 last_start_time = 3;
}